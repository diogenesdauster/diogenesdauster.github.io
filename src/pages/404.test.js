import React from 'react'
import renderer from 'react-test-renderer'
import NotFoundPage from './404.js'

/* eslint-disable */
describe('404', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<NotFoundPage data={sideData} />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
/* eslint-disable */
