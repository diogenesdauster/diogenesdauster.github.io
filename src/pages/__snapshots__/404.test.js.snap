// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `
Array [
  <div
    className="sidebar"
  >
    <div
      className="container sidebar-sticky"
    >
      <div
        className="sidebar-about"
      >
        <h1>
          <a
            href="/"
          >
            <span
              aria-label="emoji"
              role="img"
            >
              üë®üèª‚Äçüíª
               
            </span>
            <span
              className="tags"
            >
              &lt;/
            </span>
            D.Dauster
            <span
              className="tags"
            >
              &gt;
            </span>
          </a>
        </h1>
        <p
          className="tagline"
        >
          Code Has No Accent
        </p>
      </div>
      <input
        id="menu-icon"
        type="checkbox"
      />
      <label
        className="menu-label"
        htmlFor="menu-icon"
      />
      <div
        className="trigger"
      >
        <ul
          className="sidebar-nav"
        >
          <li>
            <Link
              className="sidebar-nav-item"
              to="/"
            >
              Home
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="about"
            >
              About
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="archive"
            >
              Archive
            </Link>
          </li>
          <li>
            <Link
              className="sidebar-nav-item"
              to="projects"
            >
              Projects
            </Link>
          </li>
        </ul>
      </div>
      <p
        className="copyright"
      >
         
        ¬© 
        2018
        . All rights reserved.
      </p>
    </div>
  </div>,
  <div
    className="content container"
  >
    <div
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <img
        alt="404"
        src="/images/one-punch-man.png"
      />
      <h1>
        404 Page not Found
      </h1>
    </div>
  </div>,
]
`;
